### PR TITLE
Update description of calendar in README

### LINKED ISSUE #2: Conceptual illustration of calendar
@geek-yang and I came up with a new conceptual illustration of the calendar, particularly emphasizing the important role of the anchor date/year. It's intended for the blog post we're prepping, but I think we could also use it in the README/documentation:

image

### PR DESCRIPTION
**Pull request by Peter9192**
This PR updates the description of how the calendar works in the README. It adds the conceptual figure and hence closes #2.

(I desperately wanted to contribute something other than a review... 😅)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,7 +83,13 @@ Now, the user can load the data `input_data` (e.g. `pandas` `DataFrame`) and res
 3        2021           1  [2021-11-30, 2022-05-29)      460.5   False
 ```
 
-Depending on data preparations, we can choose different types of calendars e.g. [`MonthlyCalendar`](https://ai4s2s.readthedocs.io/en/latest/autoapi/lilio/time/index.html#lilio.MonthlyCalendar) and [`WeeklyCalendar`](https://ai4s2s.readthedocs.io/en/latest/autoapi/lilio/time/index.html#lilio.WeeklyCalendar).
+For convenience, Lilio offers a few shorthands for standard of calendars e.g.
+[`monthly_calendar`](https://lilio.readthedocs.io/en/latest/autoapi/lilio/calendar_shorthands/index.html#lilio.calendar_shorthands.monthly_calendar)
+and
+[`weekly_calendar`](https://lilio.readthedocs.io/en/latest/autoapi/lilio/calendar_shorthands/index.html#lilio.calendar_shorthands.weekly_calendar).
+However, you can also create very complex calendars by calling
```
Comment (BSchilperoort) [inline README.md:90]:
```suggestion
However, you can also create custom calendars by calling
```
As `Calendar` can also be used to make simple calendars.

File: README.md [hunk 2]
```diff
@@ -72,7 +83,13 @@ Now, the user can load the data `input_data` (e.g. `pandas` `DataFrame`) and res
...
+[`Calendar`](https://lilio.readthedocs.io/en/latest/autoapi/lilio/calendar/index.html#lilio.calendar.Calendar)
+directly.
```
Comment (BSchilperoort) [inline README.md:92]: Perhaps we should refer to the `all_about_the_calendar` notebook here as well (as that feels more explanatory than the docstring).

**Review by BSchilperoort — APPROVED** [review]
Thanks for taking up this task! I left a few small comments.

File: README.md (hunk 2 above)
Comment (Peter9192) [reply README.md:92]: Good point! It would be nice to include these example notebooks in the documentation.

Comment (Peter9192) [reply README.md]: Updated the PR with a link to the repo for now.

File: README.md [hunk 3]
```diff
@@ -47,10 +47,21 @@ hatch run test
...
 ## How the lilio calendars work
-In a typical ML-based timeseries analysis, the first step is always data processing.  A calendar-based datetime module `time` is implemented for time operations. For instance, a user is looking for predictors for winter climate at seasonal timescales (~180 days). First, a `calendar` object is created using `AdventCalendar`:
+
+In Lilio, calendars are 2-dimensional. Each row (year) represents a unique
+observation, whereas each column corresponds to a precursor period with a
+certain lag. This is how we like to structure our data for ML applications.
+
+![Conceptual illustration of Lilio Calendar](docs/assets/images/calendar_concept.png)
+
+We define the "anchor date" to be between the target and precursor periods. All
```
[the remark this answers is not in the document]
Comment (Peter9192) [reply README.md]: Fair enough, I've updated that

File: README.md (hunk 2 above)
Comment (Peter9192) [reply README.md]: Opened https://github.com/AI4S2S/lilio/issues/23

**Review by BSchilperoort — APPROVED** [review]
Looks good. Thanks for taking up this part 😄

**Pull request merged**